.circleci/continue_config.yml

```diff
@@ -200,8 +209,8 @@ jobs:
       name: docker
       python_version: '3.8'
     steps:
-      - checkout
-      - install_node_dependencies
+      - setup
```

Comment:
With this step, Python dependencies are now installed, and the whole React application gets built. I'm not sure that's needed to run the Cypress e2e tests, or is it?

Reply:
For e2e tests we start both frontend and backend which requires all the dependencies to be installed and `make run` needs `make build` (which is part of npm_build in circleci setup command) to be executed.

Reply:
Right. Do we actually need to build the React application for the tests to run though? Can that be separated out? I'm asking because that step takes a lot of time.

Reply:
yea even I observed that. I also wonder why we need to make a react build to start a backend server. Let me check on that. Thank you !